config/glib.mpb

```diff
@@ -10,9 +10,21 @@ project {
     lib
   }
 
-  includes      += $(GLIB_ROOT)/include/glib-$(GLIB_VERSION) \
-                   $(GLIB_ROOT)/$(GLIB_LIB_DIR)/glib-$(GLIB_VERSION)/include
   libpaths      += $(GLIB_ROOT)/lib
 
   lit_libs      += glib-$(GLIB_VERSION)
 }
+
+feature(glib_versioned_includes) {
+  includes += $(GLIB_ROOT)/include/glib-$(GLIB_VERSION) \
+              $(GLIB_ROOT)/$(GLIB_LIB_DIR)/glib-$(GLIB_VERSION)/include
+}
+
+feature(!glib_versioned_includes) {
+  includes += $(GLIB_ROOT)/include
+  libpaths -= $(GLIB_ROOT)/lib
+  specific {
+    Debug::libpaths += $(GLIB_ROOT)/debug/lib
```

Comment:
Will this work with any OS/package manager users have? For xercesc/opendds/zlib we have the ability that the user can specify the paths to be used, as we use in the github action CI, see https://github.com/DOCGroup/ACE_TAO/blob/c22e1496f305f0fef210445f3e371b1c987ba318/.github/workflows/windows.yml#L87. Isn't that a better solution than trying to set `debug/lib` here

Reply:
I think it will work with the package managers I know of, which is a smaller set than "any."

The previous version of this .mpb works for Linux system packages and how Wireshark used to package its own dependencies for Windows (in `wireshark-win64-libs-3.0` for example).  My best guess is that this what glib does when built from source so in a sense it's the "correct" layout.  It may have been mingw so the debug/release distinction isn't there.

Newer versions of Wireshark use vcpkg for some libraries, including glib, and put the resulting vcpkg `installed` directory in `wireshark-win64-libs-3.4`.  Essentially vcpkg is making a breaking change and they expect downstream users to either be CMake or somehow incorporate MSBuild rules (I haven't learned how that works yet).

So I had two goals for this change:
- maintain existing functionality
- support Debug/Release